controllers/amphoracontroller_controller.go

```diff
@@ -463,19 +442,38 @@ func (r *OctaviaAmphoraControllerReconciler) generateServiceConfigMaps(
 	if err != nil {
 		return err
 	}
+
+	parentOctaviaName := amphoracontrollers.GetOwningOctaviaControllerName(
+		instance)
+	serverCAPassSecretName := fmt.Sprintf("%s-ca-passphrase", parentOctaviaName)
 	caPassSecret, _, err := secret.GetSecret(
-		ctx, helper, instance.Spec.CAKeyPassphraseSecret, instance.Namespace)
+		ctx, helper, serverCAPassSecretName, instance.Namespace)
 	if err != nil {
+		if k8s_errors.IsNotFound(err) {
+			instance.Status.Conditions.Set(condition.FalseCondition(
+				condition.InputReadyCondition,
+				condition.RequestedReason,
+				condition.SeverityInfo,
+				condition.InputReadyWaitingMessage))
+			return fmt.Errorf("OpenStack secret %s not found", instance.Spec.Secret)
```

Comment:
Sorry, just one last thing.  Maybe the error message here should say something about the specific server CA passphrase secret instead of a generic "OpenStack secret".

Reply:
Yes, you're right. Thanks.